rep-I0001.rst

```diff
@@ -156,27 +158,40 @@ The motion interface can be expressed as four variations:
 
-        topic_list:
-         - name: <topic name>
+        controller_joint_map:
+         - group: <controller group#>
            ns: <topic namespace>
-           group: <controller group#>
            joints:
             - <joint_1>
             - <joint_2>
             - <joint_N>
-         - name: <topic name>
+         - group: <controller group #>
            ns: ...
 
+This structure allows users to configure the robot interface to receive motion commands in several different ways:
+
+#. Single Topic
+    all groups use same namespace (e.g. `ns: ""`)
+#. Separate Topics
+    use group-specific namespaces
+#. Mixed Mode
+    joint map may contain multiple entries for the same group, in different namespaces.  This enables separate multi-arm topics for: left, right, both.
+  
+The interface node will always listen for motion commands on the same topic as the current interface (`joint_path_command`), inside the specified namespace for each group.
+
 State Interface
 =========
-The robot state interface encapsulates all the data coming FROM the robot controller, including joint position, velocity (if available), effort(if available), position error and general robot status information [#rbt_stat]_.  The implementation of the state interface is simpler than the motion interface because it can be generalized to the multi-arm case, where a single arm is just a specific example.
+The robot state interface encapsulates all the data coming FROM the robot controller, including joint position, velocity (if available), effort(if available), position error and general robot status information [#rbt_stat]_.
 
 The state interface is split into a joint state and robot status interface (although they will utilize the same socket connection, see `Communications Method`_).  The split allows joint state feedback to be sent at a higher rate than status information (which should change slowly).
- * Joint State - A single controller message is split into N JointState messages.
- * Robot Status - A single controller message that contains status information for each arm.
+
+As in the motion interface, users can specify which namespace to use for each robot group.  Unlike the motion interface, multiple groups that share a namespace are *always* combined into a single message (regardless of sync vs. async motion).  Separate namespaces can be used to receive separate messages for each motion group.
+
+For `JointState` and `JointTrajectoryFeedback` messages, aggregation of multiple robot groups is easily supported through the explicit joint-name list in the message.  For `RobotStatus` messages, the status of multiple groups is aggregated into a single message using simple logic and the `TriState` message type:  If all groups share the same status value, use that value, otherwise use `TriState::Unknown`.  Some fields (such as `in_motion`) may use modified logic (if "any" group in motion...).
```

Comment:
Also: not too sure about the "Some fields may use modified logic". Can 'a group' be said to be in motion if one or more are not?